backend/src/main/java/de/bund/digitalservice/ris/norms/domain/entity/MetadatenDs.java

```diff
@@ -109,4 +111,32 @@ public Optional<String> getSingleElementSimpleMetadatum(
         .flatMap(
             node -> new Einzelelement(node).getFrameSimpleMetadatum(metadatumSingleElement, date));
   }
+
+  /**
+   * It returns the value for a specific metadata from a single element referenced by its eid for a
```

Comment:
This looks like a copy&pasta issue ;) Could you write a better description?

Reply:
Whoopsi. Good catch, thanks!